examples/cc26xx/cc26xx-web-demo/cc26xx-web-demo.c

```diff
@@ -966,6 +992,56 @@ PROCESS_THREAD(cc26xx_web_demo_process, ev, data)
 
   PROCESS_END();
 }
+
+PROCESS_THREAD(adc_process, ev, data)
+{
+  PROCESS_BEGIN();
+  static struct etimer et_adc;
+  while(1)
+  {
+	  etimer_set(&et_adc, CLOCK_SECOND*5);
+	  PROCESS_WAIT_EVENT(); 
+	  if(etimer_expired(&et_adc)) {
```

Comment:
If you'd use PROCESS_WAIT_EVENT_UNTIL you could get rid of one level of intendation :)